__exekutir__/inventory/host_vars/kommandir.yml

```diff
@@ -87,4 +87,6 @@ git_cache_args:
     - repo: "https://github.com/python-bugzilla/python-bugzilla.git"
       recursive: False
       cachepath: "python-bugzilla"
+      # Do not include a 'depth' argument, git needs the full history to be 
+      # able to locate specific commits/branches/tags referenced by 'version'
```

Comment:
Comments are always most helpful when they're close to the relevant code. But again, I don't think this is worth spending time on right now.